src/index.ts

```diff
@@ -86,18 +88,20 @@ export default class implements ConnectPlugin {
             });
 
             if (matchedStory) {
-                links.push(this.createLink(matchedStory.kind, matchedStory.name));
+                const { storyId, kind, name } = matchedStory;
+                links.push(this.createLink({ storyId, kind, name }));
             }
         }
 
         const sbManualConf = componentConfig.storybook;
```

Comment:
```suggestion
        const { kind, stories } = componentConfig.storybook || {};
```

Reply:
I cannot destruct it since `componentConfig.storybook` is `StorybookComponentConfig | undefined`

Reply:
I think you can destruct if you add `|| {}`. WDYT?